src/Symfony/Component/VarExporter/Hydrator.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\VarExporter;
+
+use Symfony\Component\VarExporter\Internal\Hydrator as InternalHydrator;
+
+/**
+ * A utility class to hydrate objects' properties.
```

Comment:
```suggestion
 * A utility class to hydrate objects properties.
```

While it only hydrates many properties of one object?

Reply:
I meant "properties of objects", which is what "objects' properties" means, isn't it?

Reply:
reworded